cypress/integration/page.spec.js

```diff
@@ -975,7 +1053,7 @@
 
     it('should show only email error if email is empty', () => {
       newCommentForm.nameInput().type('Some name');
-      newCommentForm.bodyArea().type('Some comment body')
+      newCommentForm.bodyArea().type('Some comment body');
```

Comment:
There is a missing semicolon at the end of the `cy.type` statement. Ensure that each statement is properly terminated to avoid syntax errors.